CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ We'll do everything we can to help you get started.
 The two best ways to get help are to
 
 * jump into the [support chat](https://gitter.im/exercism/support).
-* open a [GitHub issue](https://github.com/exercism/exercism.io/issues).
+* open a [GitHub issue](https://github.com/exercism/problem-specifications/issues).
```

Comment:
I think you might have missed this bit at the top of this file:

> This guide covers several common scenarios pertaining to improving the language tracks themselves. There are other guides about contributing to other parts of the Exercism ecosystem. As such, the links are not specific to problem-specifications.

```suggestion
* open a [GitHub issue](https://github.com/exercism/exercism/issues).
```

I do agree that this is weird, and we'll likely move much of this content to our docs at some point.